afkak/test/test_consumer.py

```diff
@@ -805,9 +805,7 @@ def test_consumer_offset_out_of_range_error_with_auto_reset_to_earliest(self):
         f = Failure(OffsetOutOfRangeError())
         reqs_ds[0].errback(f)
 
-        # Check fetch 'earliest' offset request called
-        fetch_request = OffsetRequest(topic=topic, partition=part, time=OFFSET_EARLIEST, max_offsets=1)
-        consumer.client.send_offset_request.assert_called_once_with([fetch_request])
+        self.assertEqual(consumer._fetch_offset, OFFSET_EARLIEST)
 
```

Comment:
You might want to also assert in both these tests that the retry_fetch() is setup when these failures are encountered.